frontend/src/components/FooterComponent.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <footer class="footer">
+    <div class="container">
+      <div class="footer-section">
+        <img
+          src="../assets/logo-nconnect.png"
```

Comment:
Nebudeme používať relatívne cesty k assetom, pretože ak tento komponent presunieme počas vývoja niekam inam, budeme musieť meniť aj cesty.

Keď sme robili frontend projekt, vo Vite configu bolo definované že priečinok `/src` sa píše ako `@/` a za tým cesta, takže by to bolo:

```suggestion
          src="@/assets/logo-nconnect.png"
```

...a to by bola absolútna cesta a je to niečo ako "best-practice".

Keďže zatiaľ všade používame relatívne cesty, tak v tomto PR to nevadí ale bude treba vytvoriť ďalší PR kde sa definuje `@` a zmeniť existujúce cesty (to spravím ja)